atoma-service/src/streamer.rs

```diff
@@ -342,6 +342,24 @@ impl Streamer {
 
         self.is_final_chunk_handled = true;
 
+        CHAT_COMPLETIONS_STREAMING_LATENCY_METRICS.record(
```

Comment:
This only records the total duration of the decoding phase. Let's also add metrics for this on time to first token and inter token latency.